config/config.exs

```diff
@@ -34,4 +34,9 @@ config :canary,
   repo: CoursePlanner.Repo,
   unauthorized_handler: {CoursePlanner.Helper, :handle_unauthorized}
 
+config :course_planner, CoursePlanner.Scheduler,
+  jobs: [
+    {"0 18 * * *", {CoursePlanner.Notifications, :send_all_notifications, []}}
```

Comment:
i am not sure how does this work. in case the server go to sleep and wakes up again does this scheduler runs immediately ? ( if so everything is alrighty)